org.mixedrealitytoolkit.spatialmanipulation/Solvers/Solver.cs

```diff
@@ -265,6 +281,13 @@ protected virtual void Start()
             {
```

Comment:
I think you were right earlier :)  we need branching for the old rig to work.   I think this is a case where you can't remove this.  Surround with `#pragma warning disable`  .  

Also move the `controllerLookup` assignment after the `trackedPoseDriverLookup` assignment, and only if `trackedPoseDriverLookup` is null as well.

```cs
#pragma warning disable CS0618 // Type or member is obsolete
  // Find the controller lookup class in the hierarchy. Solvers that require access to the
  // left, right or gaze controllers will use the references stored in this class.
if (controllerLookup == null && trackedPoseDriverLookup == null)
{
    controllerLookup = ComponentCache<ControllerLookup>.FindFirstActiveInstance();
}
#pragma warning restore CS0618
```

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/749/commits/2e7783a6407a5447c9ecefb943a266855b096514 and https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/749/commits/7d515a9520774e8b9be79bdd22ff5d7a59bf329c , resolving.